squirrel_object_perception/src/squirrel_recognize_objects.cpp

```diff
@@ -24,8 +23,10 @@
 #include "mongodb_store/message_store.h"
 #include <squirrel_object_perception_msgs/SegmentInit.h>
 #include <squirrel_object_perception_msgs/SegmentOnce.h>
+#include <squirrel_object_perception_msgs/BBox.h>
+#include <v4r_object_recognition_msgs/recognize.h>
```

Comment:
```
fatal error: v4r_object_recognition_msgs/recognize.h: No such file or directory
 #include <v4r_object_recognition_msgs/recognize.h>
                                                   ^
```
missed dependency, where can be found this package?

Reply:
They are part of our v4r ros wrappers. I changed CMakelists.txt and package.xml